pkgs/by-name/am/amiberry/package.nix

```diff
@@ -0,0 +1,91 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  makeWrapper,
+  flac,
+  libmpeg2,
+  libmpg123,
+  libpng,
+  libserialport,
+  portmidi,
+  SDL2,
+  SDL2_image,
+  SDL2_ttf,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "amiberry";
+  version = "5.7.4";
+
+  src = fetchFromGitHub {
+    owner = "BlitterStudio";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-EOoVJYefX2pQ2Zz9bLD1RS47u/+7ZWTMwZYha0juF64=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.

More info: https://nixos.org/manual/nixpkgs/stable/#fetchurl

> Nixpkgs contributors are currently recommended to use hash.

Reply:
I swear I had changed it to `hash =`. I must have messed up with undo and some copy pasta. Good catch!